build.gradle

```diff
@@ -1,5 +1,5 @@
 group = 'com.netuitive'
-version = '0.1.24'
+version = '0.1.25'
```

Comment:
Preferably the version is only updated directly on the `develop` branch right before merging into `master`. This allows for multiple PRs to be compiled under a single release.